v0/destinations/ga4/transform.js

```diff
@@ -129,6 +130,15 @@ const responseBuilder = (message, { Config }) => {
         payload
       );
     }
+
```

Comment:
Don't we have to add `GA4_PARAMETERS_EXCLUSION` here in the exclusionFileds?

Reply:
getGA4ExclusionList() is handling it